docs/mp/grpc/server-side-services.adoc

```diff
@@ -243,15 +267,15 @@ different method signatures for a bidirectional method so the only supported sig
 public StreamObserver<RequestType> invoke(StreamObserver<ResponseType> observer)
 ----
 
-== Deploying Protobuf Services
-Whilst the examples above show how simple it is to write gRPC services with basic POJOs there may be cases where there is a
+=== Deploying Protobuf Services
+Whilst the examples above show how simple it is to write gRPC services with basic POJOs, there may be cases where there is a
```

Comment:
The examples above show how simple it is to write gRPC services with basic POJOs. There may be cases, however, where there is a